src/plugins/intel_gpu/src/graph/program_helpers.cpp

```diff
@@ -114,6 +114,7 @@ add_fusing_type onednn_add_fusing_helpers::get_add_fusing_type(
             && p_layout.format == d_layout.format && p_layout.get_tensor() == d_layout.get_tensor()
             && p_layout.data_padding == d_layout.data_padding
             && dep_node.get_users().size() == 1
+            && !dep_node.is_constant()
```

Comment:
What if the input node is not a constant, but will be a constant as the result of constant propagation? In that case, first execution of this function(before constant propagation) will return "not a constant". isn't it?

Reply:
This function is used after constant propagation pass. So, there is no intermediate node between eltwise and constant.